command/results_gotest_parser.go

```diff
@@ -109,19 +110,13 @@ func (vp *goTestParser) Logs() []string {
 
 // Results returns an array of test results parsed during test execution.
 func (vp *goTestParser) Results() []*goTestResult {
```

Comment:
the decision to have an exported method return a private type is... peculiar at best, but it certianly wasn't your fault. 

I'd err towards making the methods private, but its probably worth not touching at this venture.

Reply:
I'd prefer to not touch this right now, but these definitely should all be private